x-pack/filebeat/docs/inputs/input-azure-blob-storage.asciidoc

```diff
@@ -26,7 +26,7 @@ even though it can get expensive with dealing with a very large number of files.
     describing said error.
 
 [id="supported-types"]
-NOTE: NOTE: Currently only `JSON` and `NDJSON` are supported blob/file formats. Blobs/files may be also be gzip compressed.
+NOTE: NOTE: Currently only `JSON`, `NDJSON` and `CSV` are supported blob/file formats. Blobs/files may be also be gzip compressed.
```

Comment:
Do we need  "NOTE: NOTE:"‽

```suggestion
`JSON`, `NDJSON` and `CSV` are supported blob/file formats. Blobs/files may be also be gzip compressed.
```

Reply:
Wanted to highlight this since users could potentially miss this if not highlighted via a NOTE.

Reply:
Still one to go. In this context, I don't believe that the NOTE: will make any difference to whether a reader will pay attention to the information. If you want to ensure it's noticed, I'd suggest giving it a heading. The use of "NOTE" marks something as incidental but something to consider. If it's more important than that, which it appears to be according to what you've said, then it needs a section.